spec/controllers/documents/document_upload_question_controller_spec.rb

```diff
@@ -36,4 +36,24 @@
       end
     end
   end
+
+  describe ".show?" do
+    context "for intakes with 211 source" do
+      let(:intake) { create :intake, source: "211intake" }
+
+      it "returns false" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "for intakes with any other source" do
+      let(:intake) { create :intake }
+
+      it "raises an error because it needs the document_type method ot be defined" do
+        expect {
+          described_class.show?(intake)
+        }.to raise_error NotImplementedError
+      end
+    end
+  end
```

Comment:
I don't understand this test. It does not set another source and it implies raising an error is the correct behavior here, but I don't understand why it is the correct behavior.

Reply:
it's because this is a test for a controller that is always a parent and expects the inheriting controller to implement `document_type` - asheesh made a good point that it would be better to define a test class instead of just testing the document upload question controller itself so we did that